_studio/shared/asc/Android.mk

```diff
@@ -1,22 +1,88 @@
 LOCAL_PATH:= $(call my-dir)
 
+# =============================================================================
+
+include $(CLEAR_VARS)
+include $(MFX_HOME)/android/mfx_defs.mk
+
+LOCAL_SRC_FILES := $(addprefix src/, $(notdir $(wildcard $(LOCAL_PATH)/src/asc_avx2_impl.cpp)))
```

Comment:
here and below...

I don't see the reason to use wildcards expression here when you can just explicitly specify full path to the file. I think this will be simpler.